### PR TITLE
GW-26: Develop footer

### PR DESCRIPTION
**Pull request by chrishadrian**
Well.. it's a footer.
Also added a paper plane decoration and a scroll to top button.
Enjoy

Deploy preview: https://deploy-preview-28--gisaubcv3.netlify.app/

### CODE REVIEW

File: src/assets/footer/spotify-icon.svg [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<svg width="31" height="31" viewBox="0 0 31 31" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (Julian-UBC) [inline src/assets/footer/spotify-icon.svg:1]: I think its better to use MUI icon instead.

**Review by Julian-UBC — CHANGES_REQUESTED** [review]
image
Can you fix the responsiveness @chrishadrian. The font size and icon are not consistent.

File: src/assets/footer/spotify-icon.svg (hunk 1 above)
Comment (chrishadrian) [reply src/assets/footer/spotify-icon.svg:1]: Spotify doesn't have MUI icon. I'll change the width and height to percentage

Comment (chrishadrian) [reply src/assets/footer/spotify-icon.svg:1]: Btw responsiveness fixed for tablet view.

File: src/pages/home/components/footer/Footer.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            </div>
+            <div className="footer mt-36 lg:mt-0">
+                <div className="footer-title text-lg sm:text-2xl lg:text-3xl">Connect with us!</div>
+                <div className="footer-icons">
+                    <a href='https://www.instagram.com/gisaubc/' className="icon"><img src={instagramIcon} alt="instagram-icon" /></a>
+                    <a href='https://ca.linkedin.com/company/gisau' className="icon"><img src={linkedinIcon} alt="linkedin-icon" /></a>
+                    <a href='https://www.facebook.com/gisaubc/' className="icon"><img src={facebookIcon} alt="facebook-icon" /></a>
+                    <a href='https://open.spotify.com/show/4n3LXi2mKxLpscsIGVAgnR?si=rwRFIioeRnKj7irYurgZsQ&app_destination=copy-link' className="icon"><img src={spotifyIcon} alt="spotify-icon" /></a>
+                </div>
+                <div className="footer-email text-base sm:text-xl lg:text-2xl">
```
Comment (Julian-UBC) [inline src/pages/home/components/footer/Footer.js:24]:
```suggestion
                <div className="footer-email text-base sm:text-xl">
```

File: src/pages/home/components/footer/Footer.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+                    <a href='mailto:contact.gisau@gmail.com'><Email className='mr-1' />
+                        contact.gisau@gmail.com</a>
+                </div>
+                <div className="footer-copyright text-base sm:text-xl lg:text-2xl">GISAUBC &#169; 2022</div>
```
Comment (Julian-UBC) [inline src/pages/home/components/footer/Footer.js:28]:
```suggestion
                <div className="footer-copyright text-base sm:text-xl">GISAUBC &#169; 2022</div>
```

Similar to the other two, I feel like the `lg` one is a bit too big relative to the sizes of the content above

File: src/pages/home/components/footer/Footer.css [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+    font-weight: 700;
+    margin: 1.5rem 0 0.2rem;
+}
+
+.footer-icons {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 0.6rem 0;
+    width: 10%;
```
Comment (Julian-UBC) [inline src/pages/home/components/footer/Footer.css:23]: I think this is one of the cause for the icon size bug

File: src/pages/home/components/footer/Footer.css [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+    margin: 0.5rem 0;
+}
+
+@media only screen and (max-width: 639px) {
+    .footer-title {
+        margin-top: 1rem;
+        margin-bottom: 0.4rem;
+    }
+    .footer-icons {
+        width: 32%;
```
Comment (Julian-UBC) [inline src/pages/home/components/footer/Footer.css:60]: The other style which might cause the icon bug

File: src/pages/home/components/footer/Footer.css [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+    }
+}
+
+@media only screen and (min-width: 639px) and (max-width: 1039px) {
+    .footer-title {
+        margin-top: 1rem;
+        margin-bottom: 0.4rem;
+    }
+    .footer-icons {
+        width: 20%;
```
Comment (Julian-UBC) [inline src/pages/home/components/footer/Footer.css:75]: This is too (the icon bug)

**Review by Julian-UBC — CHANGES_REQUESTED** [review]
### Good job on the Scroll Button!! Good stuff.

image
image

Those two are some of the examples where some screen size has a bug where the icon is too big or too small compared to the other footer content.

**Review by jenaarianto — APPROVED** [review]

**Pull request merged**